articles/101_build_tool.md

```diff
@@ -210,9 +202,15 @@ The tool performs an "isolated" build like `catkin_make_isolated` and `catkin_to
 - Python package with a `package.xml` file.
 - Python package without a manifest file (extracting the package name and dependencies from the `setup.py` file).
 
+### colcon
+
+When the first draft of this article was written the conclusion was to not to spend any resources towards a universal build tool.
+As a consequence the author of this article went ahead and developed [colcon](https://github.com/colcon/) as a personal project.
+Therefore its feature set is closely aligned with the following requirements.
```

Comment:
Should we highlight here that this doesn't have usability features of catkin_tools either such as profiles (like it's been mentioned above for ament_tools)?

Reply:
It is already mentioned [above](https://github.com/ros2/design/pull/168/files#diff-361c5cb5b98ec5e7f690f2dbce2cd7fdR195) and [below](https://github.com/ros2/design/pull/168/files#diff-361c5cb5b98ec5e7f690f2dbce2cd7fdR395).

Reply:
I think that's my point... You added it above for `ament_tools` and not here while it applies to both.

The fact the selecting colcon requires that feature to be listed in the "Future" section doesn't justify not mentioning it here in my opinion.

Reply:
Please feel free to commit any proposal you think makes it better.